### PR TITLE
Add 0.11.3 release note

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Add 0.11.3 release note

## Issue reference

n/a

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by artursouza — APPROVED** [review]

**Review by artursouza — APPROVED** [review]

File: docs/release_notes/v0.11.3.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+  
+# Dapr 0.11.3
+
+## Fixes
```
Comment (msfussell) [inline docs/release_notes/v0.11.3.md:4]: Can we start with a scenario summary

File: docs/release_notes/v0.11.3.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* Improve actor service health check relibility to avoid unexpected placement disconnection(https://github.com/dapr/dapr/pull/2292).
+* Do not track Actor ID for actor pending lock count metric(https://github.com/dapr/dapr/pull/2295)
+
+### Overview
+
+We have run load test for actor to evaluate the performance and relibility with the load test since the last milestone. The test discovered [the below issues](https://github.com/dapr/dapr/issues/2093) causing memory leak:
+
+1. [Fixed in this hotfix] Frequent reconnections to placement service when app http endpoint is unresponsive intermitently under high load scneario - Fixed, 0.11.3 hotfix
+2. [Fixed in this hotfix] Memory increases from http middleware when recording request metric - Fixed, 0.11.3 hotfix
+3. No RSS memory size reclaim in Kubernetes even after actors are deactivated - Expected, documentation improvement in 1.0 RC1
```
Comment (msfussell) [inline docs/release_notes/v0.11.3.md:15]: What is RSS?

File: docs/release_notes/v0.11.3.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+4. Unnecessary memory allocation in actor invocation flow - WIP, 1.0 RC1
+
+The original plan was to release all fixes in 1.0 RC1, but we decided to release this hotfix addressing the first and second issues because these fixes reduce memory usage and improve the reliability of placement.
+
+Regarding RSS memory size, Daprd stores the activated actors in [sync.Map](https://golang.org/pkg/sync/#Map) and deletes actors when these actors are deactivated. Therefore, RSS memory for Darpd container will be increased when actor is activated, but the memory will not be dropped even after the actors are deactivated. User will still see the same amount of RSS memory usage because Go uses [MADV_FREE](https://github.com/golang/go/issues/23687) to release memory in linux. The memory will be reclaimed only when it hits memory limit. Please refer to [this article](https://www.bwplotka.dev/2019/golang-memory-monitoring/) to understand the memory usage of Go application. User may need to set `dapr.io/sidecar-memory-list` and `dapr.io/sidecar-memory-request` to limit the size of sidecar memory.
```
Comment (msfussell) [inline docs/release_notes/v0.11.3.md:20]: We do not need to talk about plans that changed.

File: docs/release_notes/v0.11.3.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+* Do not track Actor ID for actor pending lock count metric(https://github.com/dapr/dapr/pull/2295)
+
+### Overview
+
+We have run load test for actor to evaluate the performance and relibility with the load test since the last milestone. The test discovered [the below issues](https://github.com/dapr/dapr/issues/2093) causing memory leak:
+
+1. [Fixed in this hotfix] Frequent reconnections to placement service when app http endpoint is unresponsive intermitently under high load scneario - Fixed, 0.11.3 hotfix
+2. [Fixed in this hotfix] Memory increases from http middleware when recording request metric - Fixed, 0.11.3 hotfix
+3. No RSS memory size reclaim in Kubernetes even after actors are deactivated - Expected, documentation improvement in 1.0 RC1
+4. Unnecessary memory allocation in actor invocation flow - WIP, 1.0 RC1
```
Comment (msfussell) [inline docs/release_notes/v0.11.3.md:16]: WIP? Work in progress? Simply say, to be addressed in v1.0-rc1 release.

File: docs/release_notes/v0.11.3.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# Dapr 0.11.3
+
+## Fixes
+
+* Improve actor service health check relibility to avoid unexpected placement disconnection(https://github.com/dapr/dapr/pull/2292).
+* Do not track Actor ID for actor pending lock count metric(https://github.com/dapr/dapr/pull/2295)
+
+### Overview
+
+We have run load test for actor to evaluate the performance and relibility with the load test since the last milestone. The test discovered [the below issues](https://github.com/dapr/dapr/issues/2093) causing memory leak:
```
Comment (msfussell) [inline docs/release_notes/v0.11.3.md:11]: Suggest change this to "Actor load tests were run to evaluate the performance and relibility. The test discovered [these issues](https://github.com/dapr/dapr/issues/2093) that cause memory leaks"

Comment (youngbupark) [reply docs/release_notes/v0.11.3.md]: fixed.

File: docs/release_notes/v0.11.3.md (hunk 4 above)
Comment (youngbupark) [reply docs/release_notes/v0.11.3.md]: removed all 1.0-rc1 plan in this release note.

File: docs/release_notes/v0.11.3.md (hunk 3 above)
Comment (youngbupark) [reply docs/release_notes/v0.11.3.md]: removed.

File: docs/release_notes/v0.11.3.md (hunk 2 above)
Comment (youngbupark) [reply docs/release_notes/v0.11.3.md]: added

File: docs/release_notes/v0.11.3.md (hunk 1 above)
Comment (youngbupark) [reply docs/release_notes/v0.11.3.md]: changed the structure.

**Review by yaron2 — DISMISSED** [review]
lgtm

**Review by artursouza — APPROVED** [review]

**Review by orizohar — APPROVED** [review]

**Review by msfussell — APPROVED** [review]
Minor edits. Looks good.

**Review by yaron2 — APPROVED** [review]
lgtm

**Pull request merged**